File: OOP Lab/Lab 02/LinearSearch.java
import java.util.Scanner;
public class LinearSearch {
    public static void main (String [] args) {
        Scanner sc = new Scanner (System.in);
        System.out.print("Enter the length of the integer array: ");
        int n = sc.nextInt();
        System.out.println();
        int arr[] = new int[n];
        for (int i = 0; i < n; i++) {
            System.out.print("\tEnter a["+i+"] : ");
            arr[i] = sc.nextInt();
        }
        System.out.print("\n\tEnter element to be searched: ");
        int x = sc.nextInt();
        int pos = linearSearch(arr, x);
        if (pos == -1)
            System.err.println("404. NOT FOUND");
        else
            System.out.println(x + " found at index " + pos);
    }
    static int linearSearch (int a[], int x) {
        for (int i = 0; i < a.length; i++) {
            if (x == a[i])
                return i;
        }
        return -1;
    }
}

File: DS Lab/DS Lab programs by week/Week 1/Week1Sub/LinearSearch.c
#include<stdio.h>

void main()
{
	int arr[100], n, ele, i, pos;
	printf("Enter array size ");
	scanf("%d", &n);
	printf("Enter array ");
	for (i=0; i<n; i++)
	{
		scanf("%d", &arr[i]);
	}
	printf("Enter element to be searched ");
	scanf("%d", &ele);
	for (i=0; i<n; i++)
	{
		if (ele == arr[i])
		{
			pos = i;
			break;
		}
	}
	printf("Element is at %d", pos+1);
}	
